src/TestHelper/src/System/Diagnostics/RemoteExecutorTestBase.nouap.cs

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I think I would name this file something like ".process" instead of "nouap", to better call out what it is doing.